packages/psss/src/components/Table/CellComponents/PercentageChangeCell.js

```diff
@@ -21,11 +21,17 @@ const SuccessStyleText = styled.span`
  * Displays a value as a percentage
  */
 export const PercentageChangeCell = ({ percentageChange, className }) => {
-  if (percentageChange > 0) {
-    return <WarningStyleText className={className}>+{percentageChange}%</WarningStyleText>;
+  if (!percentageChange) {
+    return '-';
   }
 
-  return <SuccessStyleText className={className}>{percentageChange}%</SuccessStyleText>;
+  const roundedValue = Math.round((percentageChange - 1) * 100);
```

Comment:
It would be good if the reviewer can check for me that this calculation is correct. The values in the response from the server are the previous week divided by this week (eg. 1.5 or 0.5) and what I need to display is the change from last week to this week (eg. +50% or -50%).